.github/workflows/dev-java-publish-bintray.yml

```diff
@@ -0,0 +1,38 @@
+name: Dev build and publish Java distributions to Bintray
```

Comment:
Is it expected to have this file here?

Reply:
Yes, but there are many commits there, so I'm going to squash them and its file commit message will be lost. I deleted this one.